.github/workflows/tf-check-secure-estimate.yml

```diff
@@ -98,14 +96,6 @@ jobs:
       with:
         terraform_wrapper: false
 
-    # Initialize a new or existing Terraform working directory by creating initial files, loading any remote state, downloading modules, etc.
```

Comment:
ok for this remove of init and plan for estimate cost

Reply:
Added explicit init back in to ensure that it is executed withbackend = false